bot.py

```diff
@@ -28,25 +29,24 @@
 
 startTime = 0
 
-bot = discord.Client(intents=intents)
+bot = commands.Bot(command_prefix="/", intents=intents)
+
 
-tree = app_commands.CommandTree(bot)
 
 @bot.event 
 async def on_ready():  # When the bot is ready
-    tree.add_command(Bot())
-    tree.add_command(Misc())
-    tree.add_command(Minecraft())
-    tree.add_command(Web())
 
-    await tree.sync()  # Syncs the command tree
+    bot.tree.add_command(Bot())
+    bot.tree.add_command(Misc())
+    bot.tree.add_command(Minecraft())
+    await bot.load_extension("cogs.web")
```

Comment:
I see why you've changed this, but why is this different to the others, I don't understand the point?